megamek/src/megamek/common/autoresolve/converter/LowestForceAsUnit.java

```diff
@@ -77,10 +77,36 @@ public Formation convert() {
         }
         formation.setOwnerId(force.getOwnerId());
         formation.setName(force.getName());
+        formation.setStdDamage(setStdDamageForFormation(formation));
         calcSbfFormationStats();
         formation.setRole(firstNonNull(counter.top(), Role.getRole(UnitRole.SKIRMISHER)));
         formation.setStartingSize(formation.currentSize());
         return formation;
     }
 
+    private ASDamageVector setStdDamageForFormation(Formation formation) {
```

Comment:
Should probably be `getStdDamageForFormation` or `computeStdDamageForFormation`, as this method doesn't set anything, instead returning the ASDamageVector for storage by another function.